en_US.ini

```diff
@@ -135,7 +135,7 @@ Log Console = &Log Console
 Memory View... = Memory &View...
 More Settings... = &More Settings...
 Nearest = &Nearest
-Non-Buffered Rendering = &Non-Buffered Rendering (speedhack)
+Non-Buffered Rendering = &Discard Effects (Non-Buffered, speedup)
```

Comment:
Well, it's discarding buffers, but that's probably a bit more confusing.  Really, it is discarding "effects".  Sometimes those effects are exactly what is used to make the screen not flicker.

But yes, it's hard.  Still, I feel like "non-buffered rendering" is worse.

-[Unknown]